### PR TITLE
Fix mobile UI problems

### PR DESCRIPTION
**Pull request by ibrahimtelman**
closes #183 #160 #167

### CODE REVIEW

**Comment by ibrahimtelman** [comment]
@origranot Trying to fix the Prettier issue but in locale development, there is no error when I run Prettier

**Comment by ibrahimtelman** [comment]
@origranot After running Prettier there is no changed file in Git status. And also according to the error info, the "frontend/server/entry.static.js" is problematic. But I didn't touch that file.

image

image

**Comment by GuillemPM** [comment]
Don't worry about the entry.static.js file, this error is known and we are trying to fix it.
Btw, I'm not really sure if waves component should be in the layout.tsx, we only want to show it on index page.
Layout behavior is to render all the pages on the Slot component

File: frontend/src/routes/layout.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ export default component$(() => {
   return (
     <>
       <main data-theme="dracula" class="h-screen">
-        <section>
+        <section class="min-h-screen flex flex-col">
```
[the remark this answers is not in the document]
Comment (ibrahimtelman) [reply frontend/src/routes/layout.tsx:7]: Sure, I can move this part to the index page. It's for creating a flex layout and making sure waves always will be bottom of the page.

[the remark this answers is not in the document]
Comment (ibrahimtelman) [reply frontend/src/routes/layout.tsx]: Now, I took back the changes in the layout file.

**Pull request merged**